Concept_Templates/sample.md

```diff
@@ -0,0 +1,36 @@
+## Knowledge domain resources
+
+### Related
+
+https://goldbook.iupac.org/terms/view/S05451: definion text, discussion of sampling error, etc.
+
+https://en.wikipedia.org/wiki/Sample_(material): defition and history of the term.
+
+### Comments
+
+## General Concept Info
+
+### IRI: TBA
+### OWL type: TBA
+### Concept elucidation
+A sample is the most general concept for a specific representation of a material (in the EMMO sense) used in crystallographic experiments.
+
+The concept of a sample requires that expermients have been conducted on the material, or are planned to be conducted.
+If there are more than one distinct actor who performs or analyzes selections of experiments and/or apply their own 
+interpretations to recorded experimental outcomes, they consider different representations of the same material, i.e., 
+different samples. Multiple such samples of a material can exist in parallel with different relations to other 
```

Comment:
I understand the need to treat different experimental measurements of the same crystal (physical body) as being applied to different samples since any of the measurement may physically affect the crystal (e.g. damage due to radiation). However, should the same also be applied to different interpretations of the same experimental data? If actor A performs the measurement of a crystal, and actors B and C independently produce two crystal structure models from the same experimental data using different approaches, technically they would still be working with the same sample, would they not?

Alternatively, could this situation be better described by introducing a relation of some kind between different samples (e.g. "same sample").

Reply:
> If actor A performs the measurement of a crystal, and actors B and C independently produce two crystal structure models from the same experimental data using different approaches, technically they would still be working with the same sample, would they not?

We want to make ontological "is-a" relationships from the sample to things like "periodic crystal" and "aperiodic crystal", and want to avoid having contradictory "is-a" relationships to the same sample. Hence, actors B and C describe different samples of the same material that the experiment was carried out on. 

We had a discussion if something like "sample model" would be more precise, but it was dismissed on the basis of that strictly speaking almost all concepts are models.

Reply:
I agree with @vaitkus that different interpretations of the same experimental data should not mean that "the thing that was measured" is different. The simple "the thing that was measured" is not open to modelling or argumentation but is just a physical item. If the issue is wanting to be able to say both that "sample A is-a aperiodic crystal" and "sample A is-a periodic crystal" then isn't the proper ontological way to do it like "sample A is-modelled-by an aperiodic model" and "sample A is-modelled-by a periodic model". The "is-a" relation here is manifestly too strong, as we clearly can't conclude that the "thing that was measured" is one or the other.

Reply:
@jamesrhester The distinction in the phrasing in this proposal and your wording is if "sample" represents "the physical item that was measured" or a model of "the thing that was measured". 

When two actors draw contradicting conclusions from experimental outcomes from *different* experiments, it seems tricky to detangle the role of the physical differences in "the thing that was measured" and differences in the interpretations of the actors. So, the proposal in this PR aims to side-step that discussion with a concept of "sample" that leads to all these situations being phrased as: "the modelling of actor B and C describe different samples". This allows assigning properties to samples via is-a relations without running into contradictions.

If one accepts the above, one may still be concerned for the "extreme" case raised by @vaitkus where two actors come to contradictory conclusions from the exact same experimental outcomes. In the proposal of this PR, this situation would also be described as: "the modelling of actor B and C describe different samples". But, to accept this, one may need to shift away a bit from the view of "sample" referring to "the physical item" and towards a more modellistic view. Perhaps we need to change the concept name to, e.g., "sample model".

About the idea of a more physical concept of "sample" and 'is-modelled-by': My impression was that we went through quite a few ideas like this in our discussions and they were argued against. Perhaps we should bring it up in the next discussion now that we have more concrete suggestions to consider.